dgmmReview.html

```diff
@@ -0,0 +1,33 @@
+ <div class="row">
+    <div class="col-sm-4">
+        <div class="panel-body">
+            <div class="list-group" id="list-tab" role="tablist">
+                <a class="list-group-item list-group-item-action active" id="list-home-list" data-toggle="list" href="#1_dgmm" role="tab" aria-controls="home">1st DGMM: May 24-27, 2021 -  Uppsala (Sweden)    </a>
+
+            </div>
+        </div>
+     </div>
+    <div class="col-sm-8">
+        <div class="tab-content" >
+            <div class="tab-pane fade show active " id="1_dgmm" role="tabpanel" aria-labelledby="list-home-list">
+            <h5>1st DGMM: May 24-27, 2021 -  Uppsala (Sweden)</h5>
+                    <ul>
+                        <li> Web page: <a href="https://www.dgmm2021.se/">https://www.dgmm2021.se/</a></li>
+                        <li> Videos: <a href="https://www.youtube.com/playlist?list=PL61jQAsCUDRLjTQt4KU_YcivUxo1Tysi8">Youtube playlist</a></li>
+                        <li>Proceedings: <a href="https://www.springer.com/gp/book/9783030766566"> LNCS  12408</a></li>
+                        <li> <a href="https://www.dgmm2021.se/keynote-speakers/">Invited speakers</a>:
+                            <ul>
+                            <li> <a href="https://personal.us.es/majiro/"> Maria-Jose Jimenez</a> (Universidad de Sevilla, Spain):<br> <a href="https://link.springer.com/chapter/10.1007/978-3-030-76657-3_3">"On topological analysis of cells organization in biological images"</a></li>
+                            <li> <a href="https://people.cmm.minesparis.psl.eu/users/angulo/"> Jesús Angulo</a> (CMM-MINES ParisTech, France):<br> <a href="https://arxiv.org/pdf/2105.01339.pdf">"Some open questions on morphological operators and representations in the deep learning era"</a></li>
+                            <li> <a href="https://katalog.uu.se/profile/?id=N5-824"> Cecilia Holmgren</a> (Uppsala University, Sweden):<br> <a href="https://link.springer.com/chapter/10.1007/978-3-030-76657-3_2">"A unifying model for many important random trees of logarithmic height"</a></li>
+                            
+                            </ul>
+                        </li>
+                        <li> Best Student Paper award: Rémi Decelle from Université de Lorraine for the work entitled <a href="https://hal.archives-ouvertes.fr/hal-03144152/file/paper_47.pdf">"Digital Straight Segment Filter for Geometric Description"</a> (with Phuc Ngo, Isabelle Debled-Rennesson, Frédéric Mothe and Fleur Longuetaud)</li>
```

Comment:
If I am not wrong another was mentioned  (as "distinction"?) : Combining Deep Learning and Mathematical Morphology for Historical Map Segmentation Yizi Chen, Edwin Carlinet, Joseph Chazalon, Clément Mallet, Bertrand Duménieu and Julien Perret

Reply:
you're right, I will add it once they update the website to see exactly how they called it